bash/terraform/terraform_run.functions

```diff
@@ -95,30 +95,36 @@ __show_env() {
 #      }
 #
 #      # ... setting your own additional vars:
-#      #     - define custom_governance_vars() func.
-#      #     - call export_governance_vars.
-#      custom_governance_vars() {
+#      #     - define tf_custom_governance_vars() func.
+#      #     - call tf_export_governance_vars.
+#      tf_custom_governance_vars() {
 #          TF_VAR_aws_user=$(aws iam get-user --query 'User.UserName' --output text) || return 1
 #          export $TF_VAR_aws_user
 #      }
 #
-#      export_governance_vars || exit 1 # now use var.aws_user in your terraform
+#      tf_export_governance_vars || exit 1 # now use var.aws_user in your terraform
 #
-export_governance_vars() {
+tf_export_governance_vars() {
     git_vars || return 1 # will set $GIT_INFO
     export_build_url     # will set $BUILD_URL if possible
     export TF_VAR_git_user="$GIT_USER"
     export TF_VAR_git_info="$GIT_INFO"
     export TF_VAR_build_url="$BUILD_URL"
-    __run_if_exists custom_governance_vars || return 1
+    __run_if_exists tf_custom_governance_vars || return 1
+}
+
+# @desc DEPRECATED use tf_export_governance_vars instead
```

Comment:
Great idea to deprecate. Definitely we should be doing that going forwards.
However  we're lucky at the moment, as I'm only using libs in 3 places at the moment, so its easy for me to swap out.
Plus nobody else is actually using it yet. Not hipster enough, too much bash, not enough docker :)

So, this time, don't worry about keeping the old functions, just replace them.